symbols-loader.html

```diff
@@ -5,6 +5,9 @@
   ajax.onload = function(e) {
     var div = document.createElement('div');
     div.innerHTML = ajax.responseText;
+    if ({{LOADER_LOCATION_ENABLED}}) {
+      return document.getElementById('{{LOADER_LOCATION}}').appendChild(div)
+    }
     document.body.insertBefore(div, document.body.childNodes[0]);
```

Comment:
@kiwiupover  It looks like you're duplicating the insertion of the `div`, rather than changing it's location. I think you just want to make `document.body.childNodes[0]` on this line dynamic, right? Maybe adding a single config property like `loaderLocation` and having it default to 'body'?